plugin.json

```diff
@@ -2,8 +2,8 @@
     "id": "com.mattermost.mscalendar",
     "name": "Microsoft Calendar",
     "description": "Microsoft Calendar Integration",
-    "version": "0.1.0",
-    "min_server_version": "5.16.0",
+    "version": "0.1.0-beta",
+    "min_server_version": "5.24.0",
```

Comment:
@mickmister This can't be 5.24...

Reply:
Maybe 5.19.1+ for this beta? (KVstore was fixed)

Reply:
^^